resources/prosody-plugins/mod_filter_iq_jibri.lua

```diff
@@ -0,0 +1,25 @@
+local st = require "util.stanza";
+local apply_features_restriction = module:require "util".apply_features_restriction;
```

Comment:
i still see references to `apply_features_restriction` ?

Reply:
f*** my fault :( I havn't really tested that rename.